scripts/init-piwik.sh

```diff
@@ -64,48 +78,54 @@ fi
 echo "Done setting up piwik config..."
 cat $CONFIG_FILE
 chown www-data:www-data $CONFIG_FILE
-
-if [ ! -z $PIWIK_SEED_DATABASE ]; then
-  mysql --user=$DB_USER --password=$DB_PASSWORD --host=$DB_HOST \
-        --port=$DB_PORT -D $DB_NAME < /usr/share/nginx/html/config/base-schema.sql
+if [ ! -z $NEW_INSTALL ]; then
+	if [ ! -z $PIWIK_SEED_DATABASE ]; then
+	  mysql --connect-timeout=20 --user=$DB_USER --password=$DB_PASSWORD --host=$DB_HOST \
+		--port=$DB_PORT -D $DB_NAME < /usr/share/nginx/html/config/base-schema.sql
+	  if [ $? -eq 1 ]; then
+	    sleep 20
+	    mysql --connect-timeout=20 --user=$DB_USER --password=$DB_PASSWORD --host=$DB_HOST \
+		--port=$DB_PORT -D $DB_NAME < /usr/share/nginx/html/config/base-schema.sql
+	  fi
   SITE_SQL=$(cat <<EOF
-        INSERT INTO \`piwik_site\`
-        VALUES (1,
-                'Example Piwik Site',
-                'http://www.example.com',
-                '2014-11-01 12:00:00',
-                0,
-                1,
-                '',
-                '',
-                'UTC',
-                'USD',
-                '',
-                '',
-                '',
-                '',
-                'website',
-                0);
+	INSERT INTO \`piwik_site\`
+	VALUES (1,
+		'Example Piwik Site',
+		'http://www.example.com',
+		'2014-11-01 12:00:00',
+		0,
+		1,
+		'',
+		'',
+		'Europe/Paris',
```

Comment:
Default site timezone is still UTC above (SitesManager_DefaultTimezone), so I guess it doesn't matter if this example site moves to a different timezone.